src/examples/SketchExamplePage.tsx

```diff
@@ -0,0 +1,97 @@
+'use strict';
+import {Button, Linking, StyleSheet, Text, View} from 'react-native';
+import React, { useRef } from 'react';
+import {Example} from '../components/Example';
+import {Page} from '../components/Page';
+
+import RNSketchCanvas, { SketchCanvas } from '@terrylinla/react-native-sketch-canvas';
+
+export const SketchExamplePage: React.FunctionComponent<{}> = () => {
+  const exampleJsx = `import { SketchCanvas } from '@terrylinla/react-native-sketch-canvas';
+  
+function Example() {
+  const styles = StyleSheet.create({
+    container: {
+      flex: 1, justifyContent: 'center', alignItems: 'center', backgroundColor: '#F5FCFF',
+    },
+  });
+
+  return (
+    <View style={styles.container}>
+      <View style={{ flex: 1, flexDirection: 'row' }}>
+        <SketchCanvas
+          style={{ flex: 1 }}
+          strokeColor={'red'}
+          strokeWidth={5}
+        />
+      </View>
+    </View>
+  );
+}
+`;
+
+  const undoComponent = <View style={styles.functionButton}><Text style={{ color: 'white' }}>Undo</Text></View>;
+  const clearComponent = <View style={styles.functionButton}><Text style={{ color: 'white' }}>Clear</Text></View>;
+  const eraseComponent = <View style={styles.functionButton}><Text style={{ color: 'white' }}>Eraser</Text></View>;
+
+  const strokeComponent = (color: string) => (
+    <View style={[{ backgroundColor: color }, styles.strokeColorButton]} />
+  );
+
+  const strokeSelectedComponent = (color: string, index: number, changed: boolean) => (
+    <View style={[{ backgroundColor: color, borderWidth: 2 }, styles.strokeColorButton]} />
+  );
+
+  return (
+    <Page
+      title="Sketch Canvas"
+      description="Shows an example of the react-native-sketch-canvas component for sketching.">
+
+      <View >
+        <Text>For more information about the supported APIs, check out the{' '}
```

Comment:
PR just merged which adds a hyperlink section at the bottom of each component page with links to documentation. See the template page as an example. Could you add that section to this page in replacement of this?